app/src/test/java/com/example/mac/chartr/Adapter/RequestAdapterTest.java

```diff
@@ -0,0 +1,37 @@
+package com.example.mac.chartr.Adapter;
+
+
+
+import com.example.mac.chartr.adapters.RequestAdapter;
+
+
+import junit.framework.Assert;
+
+import org.junit.Test;
+
+
+
+/**
+ * Created by Michael Rush on 4/22/2018.
+ */
+
+public class RequestAdapterTest {
+    @Test
+    public void testSetRiderStatus() {
+        RequestAdapter test = new RequestAdapter(null);
+
+    }
+
+    @Test
+    public void testGetItemCount() {
```

Comment:
This should not still be in the code, we should not have useless tests and we already have tests for Request adapter that exists in dev.